package com.kotlin.bootstrap.security.exception

class UserAlreadyExistsException : RuntimeException() {

}